src/betterTableLayout.js

```diff
@@ -21,60 +103,53 @@ const betterTableLayout = () => {
         const date = row.find('td:nth-child(3)').clone(true, true);
         const weight = row.find('td:nth-child(4)').clone(true, true);
         const denominator = $(this).find('td:nth-child(5)').clone(true, true);
+
+        name.css('width', '');
+        mark.addClass('js-mark-cell');
         
         // combine mark and denominator
         if (!settings.liveModification) {
             let numerator = mark.text();
             if (!isNaN(parseFloat(numerator)) && numerator !== '') {
                 numerator = +parseFloat(numerator).toFixed(2);
             }
-            mark.html(`<input disabled>${numerator}</input> <b>/</b> <input disabled>${denominator.text()}</input>`);
-            
-            mark.find('input').css({
-                'margin': '0',
-                'width': '30pt',
-                'font-family': 'inherit',
-                'cursor': 'text'
-            });
-            mark.find('input:first').css('text-align', 'right');
-            mark.find('input:last').css('text-align', 'left');
-
-            // set mark column width
-            mark.css('width', '100px');
+            mark.html('');
+            mark.append(
+                `<input disabled min="0" value="${numerator}" />`,
+                '<b>/</b>',
+                `<input disabled min="0" value="${denominator.text()}" />`
+            );
```

Comment:
```suggestion
            mark.html(
                `<input disabled min="0" value="${numerator}" />` +
                '<b>/</b>' +
                `<input disabled min="0" value="${denominator.text()}" />`
            );
```
Looks cleaner, less function calls